boards/arm/adafruit_trinket_m0/adafruit_trinket_m0.dts

```diff
@@ -38,7 +38,7 @@
 		compatible = "pwm-leds";
 
 		pwm_led0: pwm_led_0 {
-			pwms = <&tcc0 2>;
+			pwms = <&tcc0 2 PWM_MSEC(20)>;
```

Comment:
`#pwm-cells` in the `tcc0` node should be also modified.

Reply:
fixed thx